script/sync-settings

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Wrapper script for sync_settings.rb - uses devops utilities
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SYNC_SCRIPT="$SCRIPT_DIR/sync_settings.rb"
+
+USAGE=$(cat <<-END
```

Comment:
Is this `USAGE` block the only place we're going to have documentation? Are you considering adding anything about this to the Platform Website?

Reply:
great callout, I'll add an issue to add to the [VCR docs page](https://depo-platform-documentation.scrollhelp.site/developer-docs/vcr-debugging#TestingwithVCR-RecordingVCRcassettes)

Reply:
oh, is that what this is all for?